game.py

```diff
@@ -259,21 +295,25 @@ def update_car_angles(self):
                 self.right_angle += ANGLE_SPEED
 
     def update_car_positions(self):
+        left_car_left_bound = self.lane_1 - self.car_center
+        left_car_right_bound = self.lane_2 - self.car_center
+        right_car_left_bound = self.lane_3 - self.car_center
+        right_car_right_bound = self.lane_4 - self.car_center
         # Left car
```

Comment:
Would be great to have a new line before the comment, helps make the code less clustered.

Reply:
done